### DISCUSSION TITLE
Change Log Needed?

### DISCUSSION THREAD

**Discussion by SJLennon** [codefori/vscode-ibmi#252] [Ideas]
I seem to be missing new features or improvements that keep getting added. I recently accidentally discovered the three dots that give an outline (?).  This:
![image](https://user-images.githubusercontent.com/67484051/130472018-46a47725-42d8-42f1-838e-88d0f0206f0d.png)

Not sure how a change log might look, but it would of necessity be brief and should probably, in many/most/some cases, have a link back to the documentation, if that is possible with Docsify.

I am willing to help keep the documentation current if I know of new features or changes.

Possibly there is already something in VS Code of which I am not aware.

**Comment by worksofliam** [comment]
This is a VS Code feature that shows folders, files, and the outline view for the current file - not something in our control.

With that being said, we do keep a changelog of our releases! https://github.com/halcyon-tech/vscode-ibmi/releases

**Comment by SJLennon** [comment]
Ah, thanks.  I'll troll through the releases...  It's also an incentive to learn more about Git and see the actual changes in each release.

I see 0.6.6 and 0.6.4 mention an outline.  Can you give me a quick hint on how to find it?
